docs/user_guide/expand_path.rst

```diff
@@ -71,6 +71,141 @@ If your session start time is present in your data path, you can indicate this f
 `1989 C standard format codes`_ for datetimes. For example, ``"{session_start_time:%Y-%m-%d}"`` will match
 ``"2021-01-02"`` and evaluate it to ``datetime.datetime(2021, 1, 2)``.
 
+Example Usage
+-----------------------
+
+Below are some full examples of how this feature can be used on some organizational patterns taken from real datasets.
+
+Example 1: `Allen Institute Visual Coding Dataset <https://registry.opendata.aws/allen-brain-observatory/>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Allen Institute's Visual Coding dataset contains, among other data, motion-corrected videos of each
+experimental session, with the directory structure shown below.
+
+.. code-block:: bash
+
+    allen-brain-observatory/
+    ¦   visual-coding-2p/
+    ¦   +-- ophys_movies/
+    ¦   ¦   +-- ophys_experiment_645691416.h5
+    ¦   ¦   +-- ophys_experiment_<session_id>.h5
+    ¦   ¦   +-- ...
+
+To use :py:class:`~neuroconv.tools.path_expansion.LocalPathExpander` to find all ``ophys_movies`` files and
+extract their session IDs, you could use the following code.
+
+.. code-block:: python
+
+    >>> source_data_spec = {
```

Comment:
We only use the `>>>` directive in code blocks when pairing with doctest (such as the conversion gallery); not saying there shouldn't be tests for this on each of these example structures, but utilizing doctest for it would take a little bit of doing (generating the files under the hood, adding proper imports, setting up global paths, etc)

Probably best left to follow-up PRs if desired, so I'd remove the `>>>` for now